include/bluetooth/iso.h

```diff
@@ -79,12 +79,13 @@ struct bt_iso_chan_qos {
 	/** @brief Channel direction
 	 *
 	 *  Possible values: BT_ISO_CHAN_QOS_IN, BT_ISO_CHAN_QOS_OUT or
-	 *  BT_ISO_CHAN_QOS_INOUT.
+	 *  BT_ISO_CHAN_QOS_INOUT. Shall be BT_ISO_CHAN_QOS_OUT for broadcast
+	 *  transmitting, and BT_ISO_CHAN_QOS_IN for broadcast receiver.
```

Comment:
Perhaps it would be a good idea to have separate direction for Broadcast e.g.: BT_ISO_CHAN_QOS_RECEIVER so it clearly indicates to users what kind of channel transport it is using.

Reply:
Interesting idea. I'm a bit concerned that having both a `IN` and a `RECEIVER`, as well as an `OUT` and a `BROADCASTER`, might feel like duplication for some. 

I think a better solution might be to do as you suggest below with a broadcast and a unicast struct, and simply leave out the `dir` for the broadcaster. I guess we could also simply ignore the `dir` QOS value for broadcasting, as it is very clear what the ISO will do, as there are separate APIs for broadcaster and receiver.